src/Illuminate/Foundation/Console/CastMakeCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\GeneratorCommand;
+
+class CastMakeCommand extends GeneratorCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'make:cast';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new cast mutator';
```

Comment:
"mutator" is only half of the story, as it's also for get-ting, how about
```suggestion
    protected $description = 'Create a new custom Eloquent attribute caster';
```

Reply:
Fixed